### DISCUSSION TITLE
How to set Fastly named backends for initial Hono fetch requests?

### DISCUSSION THREAD

**Discussion by alex8bitw** [honojs/hono#2727] [Q&A]
Hi @yusukebe ,
  I'm assuming this can be done through middleware, but I'm a bit hazy on how:
Doing something like this works from within an app.all():
```
import { Backend } from "fastly:backend";
 url = new URL('mozilla.org' + c.req.path);

const backend = new Backend({
  name: 'origin_host',
  target: 'mozilla.org',
  connectTimeout: 15000,
  firstByteTimeout: 15000,
  betweenBytesTimeout: 15000,
  useSSL: true
});

const newRequest = new Request(url,c.req.raw);
const res = await fetch(newRequest,{backend})
```

But how do we get the initial fetch from origin in hono to use the `explicit Dynamic Backend`:  https://js-compute-reference-docs.edgecompute.app/docs/fastly:backend/Backend/

**Comment by yusukebe** [comment]
@alex8bitw 

Hmm. I don't know, but at a glance, I feel that we don't need to use Hono in the `explicit Dynamic Backend` case.
